src/openrct2/ride/coaster/BolligerMabillardTrack.hpp

```diff
@@ -12637,25 +12637,19 @@ void BolligerMabillardTrackBooster(
     PaintSession& session, const Ride& ride, uint8_t trackSequence, uint8_t direction, int32_t height,
     const TrackElement& trackElement)
 {
-    // These offsets could be moved to the g2.dat file when that supports offsets.
-    int8_t ne_sw_offsetX = 8;
-    int8_t ne_sw_offsetY = -17;
-    int8_t nw_se_offsetX = -17;
-    int8_t nw_se_offsetY = 8;
-
     switch (direction)
     {
         case 0:
         case 2:
             PaintAddImageAsParentRotated(
-                session, direction, session.TrackColours.WithIndex(SPR_G2_BM_BOOSTER_NE_SW),
-                { ne_sw_offsetX, ne_sw_offsetY, height }, { { 0, 6, height }, { 32, 20, 3 } });
+                session, direction, session.TrackColours.WithIndex(SPR_G2_BM_BOOSTER_NE_SW), { 8, -17, height },
+                { { 0, 6, height }, { 32, 20, 3 } });
```

Comment:
The heights should always be `{ 0, 0 height }` to allign with the rest of the code, since we want to merge all drawing code at some point. If this doesn't work the allignments aren't correct in `sprites.json`.